react-native-libraries.json

```diff
@@ -9960,5 +9960,13 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/ekreative/react-native-braintree",
+    "npmPkg": "@ekreative/react-native-braintree",
+    "nameOverride": "@ekreative/react-native-braintree",
```

Comment:
Hello @vasylnahuliak, thank you for adding new library to the directory! 👍 

```suggestion
```

The package name is the default display name, so in this case override can be skipped, let's update that and we can ship the addition.